File: Problems/No.27_Remove_Element/remove_element.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        
        // position of the current index
        int cur_index = 0;
        
        // start compute
        while (cur_index != nums.size()) {
            
            if (nums[cur_index] == val) nums.erase(nums.begin()+cur_index);
            else cur_index++;
            
        }
        
        return cur_index;
        
    }
};

File: Problems/No.27_Remove_Element/remove_element.py
class Solution(object):
    def removeElement(self, nums, val):
        """
        :type nums: List[int]
        :type val: int
        :rtype: int
        """
        
        # start compute
        while (val in nums):
            nums.remove(val)
            
        return len(nums)
